karaf/pom.xml

```diff
@@ -34,21 +34,29 @@
   
   <properties>
 
+    <osgi.version>6.0.0</osgi.version>
     <org.osgi.compendium.version>5.0.0</org.osgi.compendium.version> <!-- duplicate in brooklyn-dist/dist-karaf/pom.xml -->
-    
+
     <lifecycle-mapping-plugin.version>1.0.0</lifecycle-mapping-plugin.version>
 
     <!-- pax-exam -->
-    <pax.exam.version>4.7.0</pax.exam.version>
-    <pax.url.version>2.4.3</pax.url.version>
+    <pax.exam.version>4.9.1</pax.exam.version>
+    <pax.url.version>2.5.2</pax.url.version>
     <tinybundles.version>1.0.0</tinybundles.version>
 
     <!-- feature repositories -->
     <servicemix.version>6.0.0</servicemix.version>
     <reflections.bundle.version>0.9.9_1</reflections.bundle.version> <!-- see reflections.version -->
 
+    <maven.buildhelper.plugin.version>1.10</maven.buildhelper.plugin.version>
+    <maven.resources.plugin.version>3.0.1</maven.resources.plugin.version>
+    <maven.depends.plugin.version>1.3</maven.depends.plugin.version>
+
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
+
+    <okhttp.bundle.version>2.2.0_2</okhttp.bundle.version>
```

Comment:
I wonder where this should be defined - should it be in the root `pom.xml`, alongside all the other dependency versions? Or should it be here (closer to where it's used in feature.xml)? I lean towards the former. But no strong feelings.